### PR TITLE
Update issue.html

### PR DESCRIPTION
**Pull request by diogocp**

### CODE REVIEW

File: issue.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
 <!--
 
 	---------------------
-	UNTITLED BY PIXEL LAB
+	KITSI'S GARDEN BY DORY
```
Comment (matwiild) [inline issue.html]: it's not a garden, it's an island

Comment (diogocp) [inline issue.html]: :'(

Comment (matwiild) [inline issue.html]: island -----> japan!!

Comment (diogocp) [inline issue.html]: woohoo! let's go!

Comment (matwiild) [inline issue.html]: to bed? ahaha

Comment (diogocp) [inline issue.html]: better now?

Comment (diogocp) [inline issue.html]: this is the outdated version. for the current version, look in files changed

**Pull request merged**